applications/zigbee_weather_station/README.rst

```diff
@@ -0,0 +1,519 @@
+  The value is provided in seconds, with the default value set to ``60``.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`applications/zigbee_weather_station`
+
+.. include:: /includes/build_and_run.txt
+
+Selecting a build type
+======================
+
+Before you start testing the application, you can select one of the :ref:`zigbee_weather_station_app_build_types`, depending on your building method.
+
+Selecting a build type in |VSC|
+-------------------------------
+
+.. include:: /gs_modifying.rst
+   :start-after: build_types_selection_vsc_start
+   :end-before: build_types_selection_vsc_end
+
+Selecting a build type in SES
+-----------------------------
+
+.. include:: /gs_modifying.rst
+   :start-after: build_types_selection_ses_start
+   :end-before: build_types_selection_ses_end
+
+Selecting a build type from command line
+----------------------------------------
+
+.. include:: /gs_modifying.rst
+   :start-after: build_types_selection_cmd_start
+   :end-before: For example, you can replace the
+
+For example, you can replace the *selected_build_type* variable to build the ``release`` firmware for ``thingy53_nrf5340_cpuapp`` by running the following command in the project directory:
+
+.. parsed-literal::
+   :class: highlight
+
+   west build -b thingy53_nrf5340_cpuapp -d build_thingy53_nrf5340_cpuapp -- -DCONF_FILE=prj_release.conf
+
+The ``build_thingy53_nrf5340_cpuapp`` parameter specifies the output directory for the build files.
+
+.. note::
+   If the selected board does not support the selected build type, the build is interrupted.
+   For example, if the ``shell`` build type is not supported by the selected board, the following notification appears:
+
+   .. code-block:: console
+
+      File not found: ./ncs/nrf/applications/zigbee_weather_station/configuration/thingy53_nrf5340_cpuapp/prj_shell.conf
+
+Testing
+=======
+
+Preliminary testing information
+    * The testing procedure assumes you are using the ``debug`` :ref:`build type <zigbee_weather_station_app_build_types>`, as it provides more feedback with **LED (LD1)** and logs through the USB console.
+      The steps related to the ``debug`` build type are marked as "Optional ``debug`` build type step".
+      These steps are not required when using the ``release`` build type.
+    * Wireshark with :ref:`ug_zigbee_tools_sniffer` is used for packet capturing.
+      This is an optional setup, but Wireshark response frame examples are provided in the Testing steps as reference.
+    * In the examples provided in the Testing section, Thingy:53 has the short network address ``0x1ca9`` and the long network address ``f4ce3623d74e771d``.
+      The coordinator device has the short address ``0x0000`` and the long address ``f4ce3655fc08bf6b``.
+      The coordinator's endpoint number is ``64``.
+    * Endpoint number ``42`` is used for aggregating temperature (``0x0402``), pressure (``0x0403``) and relative humidity (``0x0405``) clusters.
```

Comment:
```suggestion
```
I think this might be actually removed, as it is explained next to the examples. @kittydepa , what do you think?

Reply:
Done